gensim/models/fasttext.py

```diff
@@ -170,18 +171,10 @@ def init_ngrams(self, update=False):
 
             rand_obj = np.random
             rand_obj.seed(self.seed)
-            new_vocab_rows = rand_obj.uniform(
-                -1.0 / self.vector_size, 1.0 / self.vector_size,
-                (len(self.wv.vocab) - self.old_vocab_len, self.vector_size)
-            )
+            new_vocab_rows = rand_obj.uniform(-1.0 / self.vector_size, 1.0 / self.vector_size, (len(self.wv.vocab) - self.old_vocab_len, self.vector_size))
```

Comment:
What's the point of reformatting this and the other lines? It makes the lines too long and the code is harder to read.